cmd/deploy/config.go

```diff
@@ -39,20 +39,23 @@ func (k *kubeConfig) Modify(port int, sessionToken, destKubeconfigFile string) e
 		return err
 	}
 
+	// We should change only the config for the proxy, not the one in Context.Cfg
+	proxyCfg := clientCfg.DeepCopy()
```

Comment:
For now is ok, but I don't know if all things executed by Okteto during the `okteto deploy` command should pass also through the proxy or not. Right now I cannot think of any case in which might be needed but it would depend on how the command evolve in the future